templates/logging-sidecar-configmap.yaml

```diff
@@ -32,60 +32,12 @@ data:
           component: "${COMPONENT:--}"
           workspace: "${WORKSPACE:--}"
           release: "${RELEASE:--}"
-
-      filter_common_logs:
```

Comment:
given we don't need the broken transform below there is no need to split the logs in this way.